core_backend/app/urgency_detection/routers.py

```diff
@@ -37,37 +39,52 @@ def urgency_classifier(classifier_func: Callable) -> Callable:
 
 @router.post("/urgency-detect", response_model=UrgencyResponse)
 async def classify_text(
-    urgency_query: UrgencyQuery, asession: AsyncSession = Depends(get_async_session)
+    urgency_query: UrgencyQuery,
+    asession: AsyncSession = Depends(get_async_session),
+    user_db: UserDB = Depends(auth_bearer_token),
 ) -> UrgencyResponse:
     """
     Classify the urgency of a text message
     """
     feedback_secret_key = generate_secret_key()
     urgency_query_db = await save_urgency_query_to_db(
-        feedback_secret_key, urgency_query, asession
+        user_id=user_db.user_id,
+        feedback_secret_key=feedback_secret_key,
+        urgency_query=urgency_query,
+        asession=asession,
     )
 
     classifier = ALL_URGENCY_CLASSIFIERS.get(URGENCY_CLASSIFIER)
     if not classifier:
         raise ValueError(f"Invalid urgency classifier: {URGENCY_CLASSIFIER}")
 
-    urgency_response = await classifier(asession, urgency_query)
+    urgency_response = await classifier(
+        user_id=user_db.user_id, urgency_query=urgency_query, asession=asession
+    )
 
-    await save_urgency_response_to_db(urgency_query_db, urgency_response, asession)
+    await save_urgency_response_to_db(
+        urgency_query_db=urgency_query_db,
```

Comment:
I like this as our coding standard. Can you add it to [Ways of Working](https://docs.google.com/document/d/1fIDbGrUdBKP0qnSkVVNpwjYmB1e8FW2n9Uzei_9v7k8/edit#heading=h.k4abvzom2mqp)?

Reply:
Maybe if # of args in > 2

Reply:
Great! Requested edit access

Reply:
Added :)